packages/postcss-merge-rules/src/lib/ensureCompatibility.js

```diff
@@ -54,6 +54,7 @@ export const pseudoElements = {
   '::marker': 'css-marker-pseudo',
   '::placeholder': 'css-placeholder',
   '::selection': 'css-selection',
+  '::-moz-selection': 'css-selection',
```

Comment:
Let's put all elements and classes from https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements and https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes

Also please use http://api.postcss.org/vendor.html#.unprefixed for removing vendor prefixes

Reply:
> Let's put all elements and classes from https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements and https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes

I will try to do this for elements/classes for which the `caniuse` api is responding,  Cause for most of them, `caniuse` might have different format name, which I dont know where to look for other than requesting manually from their api.

Reply:
Set them unsupported by default, then I will look on list and give advices

Reply:
> Set them unsupported by default, then I will look on list and give advices

if they are not on the list, this PR changes are considering them unsupported.

we can even add those elements incrementally in future. Cause by default they are unsupported

Reply:
Instead of increasing the content of this PR diff, 

Can I add these elements in future PR after merging this? I will add the  `verdor.unprefixed` in this PR and rest in the next PR?

Reply:
No, it is a breaking change, we should solve it here

Reply:
ok coool